### ISSUE TITLE
Scanning e-passport throws an exception

### ISSUE THREAD

**Issue by andydixon** [L1L1/cardpeek#95]
I scanned my passport, and by the time it got to my biometrics, the following exception got thrown:

2283 ERROR   [string "e-passport.lua"]:427: variable 'BL_FEATURE_POINT' is not declared
2284 ERROR   backtrace: called from error()
2285 ERROR   backtrace: called from __index() in .\lib\strict.lua[37]
2286 ERROR   backtrace: called from ui_parse_biometry_fac() in [string "e-passport.lua"][427]
2287 ERROR   backtrace: called from tlv_ui_func() in [string "e-passport.lua"][505]
2288 ERROR   backtrace: called from internal_tag_value_parse() in .\lib\tlv.lua[250]
2289 ERROR   backtrace: called from internal_tlv_parse() in .\lib\tlv.lua[276]
2290 ERROR   backtrace: called from internal_tag_value_parse() in .\lib\tlv.lua[244]
2291 ERROR   backtrace: called from internal_tlv_parse() in .\lib\tlv.lua[276]
2292 ERROR   backtrace: called from internal_tag_value_parse() in .\lib\tlv.lua[244]
2293 ERROR   backtrace: called from internal_tlv_parse() in .\lib\tlv.lua[276]
2294 ERROR   backtrace: called from internal_tag_value_parse() in .\lib\tlv.lua[244]
2295 ERROR   backtrace: called at .\lib\tlv.lua
2296 ERROR   backtrace: called at [string "e-passport.lua"]

**Comment by ValZapod** [comment]
Is this https://github.com/L1L1/cardpeek/pull/107 fix for this issue?

**Comment by andydixon** [comment]
Yep. That's my proposed fix for it. It's worked perfectly for me, just waiting for it to hopefully be immortalised in the main codebase.
